### ISSUE TITLE
Addon not selling anything on 10.0.2.46924

### ISSUE THREAD

**Issue by robthemonster** [Beast-Masters-addons/SellJunk#2]
Addon doesn't sell anything when I open a vendor window. I see this stack trace:

```
Message: Interface/AddOns/SellJunk/SellJunk.lua:102: attempt to index local 'item' (a nil value)
Time: Tue Dec  6 20:17:15 2022
Count: 1
Stack: Interface/AddOns/SellJunk/SellJunk.lua:102: attempt to index local 'item' (a nil value)
[string "@Interface/AddOns/SellJunk/SellJunk.lua"]:102: in function `Sell'
[string "@Interface/AddOns/SellJunk/SellJunk.lua"]:78: in function `?'
[string "@Interface/AddOns/SellJunk/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:119: in function <...unk/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:119>
[string "=[C]"]: ?
[string "@Interface/AddOns/SellJunk/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:29: in function <...unk/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:25>
[string "@Interface/AddOns/SellJunk/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:64: in function `Fire'
[string "@Interface/AddOns/SellJunk/libs/AceEvent-3.0/AceEvent-3.0.lua"]:120: in function <...e/AddOns/SellJunk/libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals: <none>
```

**Comment by robthemonster** [comment]
Seems to be resolved with https://github.com/Beast-Masters-addons/SellJunk/pull/3

**Comment by datagutten** [comment]
Fixed with #3
